packages/runtime/src/templates/getHandler.ts

```diff
@@ -1,3 +1,6 @@
+// This is a runtime function so assuming we don't want to add
```

Comment:
double check, but I think you might be able to exclude the whole packages/runtime/src/templates/ folder from this eslint rule.

Reply:
Great. I'll check this.

Reply:
Can you do this for a single rule @nickytonline?
I can see how you exclude linting a set of files but not for a single rule.

Reply:
You can create overrides with globs. e.g.

https://github.com/netlify/next-runtime/blob/e1a3dc0a79a1ce388af306d0788077670507dd08/demos/nx-next-monorepo-demo/apps/demo-monorepo-e2e/.eslintrc.json#L1-L10

Reply:
You can do it for a single rule. See https://eslint.org/docs/latest/use/configure/rules#using-configuration-files-1

Reply:
Done! 👍🏻